packages/salesforcedx-visualforce-markup-language-server/src/index.ts

```diff
@@ -28,22 +24,6 @@ import { doHover } from './services/htmlHover';
 import { findDocumentLinks } from './services/htmlLinks';
 import { findDocumentSymbols } from './services/htmlSymbolsProvider';
 
-export {
```

Comment:
these were all exported from the ls-types, but other packages could just import from the ls-types repo instead of from here